src/com/twitter/intellij/pants/service/PantsCompileOptionsExecutor.java

```diff
@@ -196,8 +196,22 @@ private ProcessOutput getProcessOutput(
     return processOutput;
   }
 
+  private boolean isAtLeastPantsVersion(int major, int minor) throws ExecutionException {
+    String pantsVersion = getPantsVersion();
+    String[] splitVersion = pantsVersion.trim().split("\\.");
+    return splitVersion.length >= 3 && Integer.parseInt(splitVersion[0]) >= major && Integer.parseInt(splitVersion[1]) >= minor;
+  }
+
+  @NotNull
+  private String getPantsVersion() throws ExecutionException {
+    final GeneralCommandLine commandLine = PantsUtil.defaultCommandLine(getProjectPath());
+    commandLine.addParameter("--version");
+    return getProcessOutput(commandLine).getStdout();
+  }
```

Comment:
You can grab this from `PantsOptions` with `pants_version` as the key
```
$ ./pants options | grep pants_version
pants_version = 1.24.0.dev1 (from HARDCODED)
```
So PantsOptions should be cached already at this point, so that will save a call to Pants.

Reply:
Fixed